Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/EmulationActivity.java

```diff
@@ -828,15 +828,14 @@ public void chooseDoubleTapButton()
   private void adjustScale()
   {
     LayoutInflater inflater = LayoutInflater.from(this);
-    View view = inflater.inflate(R.layout.dialog_seekbar, null);
+    View view = inflater.inflate(R.layout.dialog_input_adjust, null);
 
-    final SeekBar seekbar = view.findViewById(R.id.seekbar);
-    final TextView value = view.findViewById(R.id.text_value);
-    final TextView units = view.findViewById(R.id.text_units);
+    final SeekBar seekbarScale = view.findViewById(R.id.input_scale_seekbar);
+    final TextView valueScale = view.findViewById(R.id.input_scale_value);
```

Comment:
Maybe it's just me, but I think `scaleSeekbar` and `scaleValue` would make more sense. Otherwise it sounds like `valueScale` is describing the scale of a value rather than the value of a scale. You can ignore this comment if you disagree.

Reply:
I mean I'm fine with it either way as long as the necessary function gets carried out.